### DISCUSSION TITLE
更新游戏版本时漏掉了游戏列表中显示的该游戏的版本号

### DISCUSSION THREAD

**Discussion by nukemiko** [Gamuxorg/bbs#359] [游戏或网站运行反馈]
最近我上传并且审核通过的两款游戏[《瘟疫公司：进化》](https://github.com/Gamuxorg/bbs/issues/171)和[《星露谷物语》](https://github.com/Gamuxorg/bbs/issues/173)，前者更新到了1.18.4，后者为1.5.6：

![Screenshot_20220305_000400](https://user-images.githubusercontent.com/53572814/156797849-2b1d2bcb-b799-48a6-8a5a-aed2a4e1d3ea.png)
![Screenshot_20220305_000436](https://user-images.githubusercontent.com/53572814/156797883-2f4e0622-d68d-4e9a-ac4a-c2d3b5cd957a.png)

但是在游戏列表中浏览时，两者仍然显示旧版的版本号（1.17.1和1.4.5）：

![Screenshot_20220305_000102](https://user-images.githubusercontent.com/53572814/156797444-9f919a7f-cf06-47e4-87a4-cf3706957067.png)
![Screenshot_20220305_000121](https://user-images.githubusercontent.com/53572814/156797483-98b96a16-7ee7-4225-b7da-f5c0c80da1a1.png)

游戏列表中显示的版本号没有更新，这样可能让新用户产生误解，建议尽快修正这一疏漏。

**Comment by hyh19962008** [comment]
感谢反馈，后台代码需要改一下

**Comment by hyh19962008** [comment]
已修复
